File: scripts/trim-page.js
const fs = require('fs');
const path = require('path');
const filePath = path.join(__dirname, '..', 'app', 'page.tsx');
const content = fs.readFileSync(filePath, 'utf8');
const lines = content.split('\n');
const keepLines = lines.slice(0, 649);
const newContent = keepLines.join('\n') + '\n}\n';
fs.writeFileSync(filePath, newContent);
console.log('Done');

File: scripts/trim_page.py
# -*- coding: utf-8 -*-
import os

script_dir = os.path.dirname(os.path.abspath(__file__))
page_path = os.path.join(script_dir, '..', 'app', 'page.tsx')

with open(page_path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Keep lines 1-649 (0-indexed: 0-648), add closing brace
new_content = ''.join(lines[:649]) + '}\n'

with open(page_path, 'w', encoding='utf-8') as f:
    f.write(new_content)

print('Done')
